lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -909,8 +909,15 @@ MStatus UsdMayaJobExportArgs::GetDictionaryFromEncodedOptions(
                         return MS::kFailure;
                     }
                 }
+
+                // Note: when round-tripping settings, some extra settings are not part
+                //       of the guiding dictionary. Ignore them.
+                const static bool reportError = false;
```

Comment:
When import/export plugin add their custom settings, this would report errors about "unknown" settings.